guides/common/modules/ref_custom-provisioning-snippet-example-for-debian.adoc

```diff
@@ -8,7 +8,7 @@ You can use `Custom Post` snippets to call external APIs from within the provisi
 ----
 echo "Calling API to report successful host deployment"
 
-apt-get install -y curl ca-certificates
+apt install -y curl ca-certificates
```

Comment:
```suggestion
{client-package-install-deb} install -y curl ca-certificates
```

as above: you have to allow attributes in the code block; you can just overwrite line 7; see comment above.